test/e2e/scalability/load.go

```diff
@@ -483,10 +486,16 @@ func GenerateConfigsForGroup(
 }
 
 func generateServicesForConfigs(configs []testutils.RunObjectConfig) []*v1.Service {
-	services := make([]*v1.Service, 0, len(configs))
+	services := make([]*v1.Service, 0)
+	currentSvcLabel := ""
 	for _, config := range configs {
+		svcLabel, found := config.GetLabelValue(svcLabelKey)
+		if !found || svcLabel == currentSvcLabel {
+			continue
+		}
+		currentSvcLabel = svcLabel
 		serviceName := config.GetName() + "-svc"
-		labels := map[string]string{"name": config.GetName()}
```

Comment:
Oops.. I added it here instead https://github.com/shyamjvs/kubernetes/blob/46b7341467125f79d6d9641b4b3af30025157df0/test/e2e/scalability/load.go#L464
Will fix it.